client/src/containers/NotRegisteredPage.tsx

```diff
@@ -42,19 +42,20 @@ export default class NotRegisteredPage extends Component<Props, State> {
 
     const { boro, block, lot } = Helpers.splitBBL(searchAddrBbl);
 
-    const usersInputAddress = searchAddrParams
-      ? {
-          boro: searchAddrParams.boro,
-          housenumber: searchAddrParams.housenumber || " ",
-          streetname: searchAddrParams.streetname,
-        }
-      : buildingInfo
-      ? {
-          boro: buildingInfo.boro,
-          housenumber: buildingInfo.housenumber || " ",
-          streetname: buildingInfo.streetname || " ",
-        }
-      : null;
+    const usersInputAddress =
```

Comment:
This is still a little confusing to me, but I guess it's just if you have both `searchAddrParams` and `searchAddrBbl` use them, otherwise if you have building info use that.

What happens if you don't have `searchAddrBbl` but you do have `buildingInfo`? Should the `: buildingInfo` bit really be `:buildingInfo && searchAddrBbl`?

Reply:
Or will that never happen?

Reply:
Oh good point! yeah this logic is unnecessarily confusing. The idea is that if the user didn't type in the address themselves from the homepage (like if they redirect from a BBL-based url), we will need to rely on the "buildingInfo" query to grab an address for the property in question as a fallback.

Reply:
I simplified the logic and also added a comment to explain